parsing/__init__.py

```diff
@@ -139,171 +139,19 @@
 from six.moves import zip
 import six.moves.cPickle
 
+from parsing.grammar import (Exception, SyntaxError, SpecError,
+    Precedence, SymbolSpec, NontermSpec, TokenSpec,
+    Production, EndOfInput, eoi, Epsilon, epsilon, NontermStart)
+from parsing.ast_objs import Symbol, Nonterm, Token
```

Comment:
These module names are short and readable, so let's avoid the use of underscores.